components/remote_settings/Cargo.toml

```diff
@@ -44,3 +47,4 @@ mockito = "0.31"
 # We add the perserve_order feature to guarantee ordering of the keys in our
 # JSON objects as they get serialized/deserialized.
 serde_json = { version = "1", features = ["preserve_order"] }
+mock_instant = "0.5.1"
```

Comment:
I'm wondering if we can implement this with less dependencies.  When this gets merged to moz-central, all the new dependencies will require an audit, so we should only add them if they're bringing in a lot of functionality.  Is there another way to get a mock time for the unit tests?

OTOH, canonical_json is easy to audit since you're the one publishing it.

Side question: how does the epoch time get used in the signatures?  How far off would a user's clock have to be to get a false negative?

Reply:
> Is there another way to get a mock time for the unit tests?

I'll try using a static/lazy

> Side question: how does the epoch time get used in the signatures? How far off would a user's clock have to be to get a false negative?

The epoch is used to validate the certificate. It is valid for several weeks and we refresh the signatures regularly to pick the latest ones. Client clock would have to be days/weeks off for a false negative